src/components/combobox/combobox.e2e.ts

```diff
@@ -62,28 +61,61 @@ describe("calcite-combobox", () => {
 
   it("can be disabled", () => disabled("calcite-combobox"));
 
-  it.skip("should filter the items in listbox when typing into the input", async () => {
+  it("should filter the items in listbox when typing into the input", async () => {
     const page = await newE2EPage({
-      html: html` <calcite-combobox>
-        <calcite-combobox-item value="one" text-label="one"></calcite-combobox-item>
-        <calcite-combobox-item value="two" text-label="two"></calcite-combobox-item>
-      </calcite-combobox>`
+      html: html`
+        <calcite-combobox id="myCombobox">
+          <calcite-combobox-item value="Raising Arisona" text-label="Raising Arisona"></calcite-combobox-item>
```

Comment:
Nitpicks:

* Arizona
* Miller's

Sorry, can't help it. 😂

Reply:
I keep accumulating English grammar enthusiasts.